src/block/interface.ts

```diff
@@ -54,7 +52,7 @@ export interface Block<
   cid: Link<T, C, A, V>
 }
 
-export type BlockCursorView<T extends unknown = unknown> =
+export type BlockCursorView<T = unknown> =
```

Comment:
was the `extends unknown` a problem here? I don't really know what it did, but Chesterton's Fence tells me to be wary of removing it (or at least check with Gozala before doing this!).

Reply:
Yeah, the linter complained:
```
Constraining the generic type `T` to `unknown` does nothing and is unnecessary.eslint[@typescript-eslint/no-unnecessary-type-constraint](https://typescript-eslint.io/rules/no-unnecessary-type-constraint)
```